vmdb/Gemfile

```diff
@@ -53,16 +53,18 @@ gem "ancestry",                       "~>1.2.4",      :require => false
 gem "aws-sdk",                        "~>1.56.0",     :require => false
 gem 'dalli',                          "~>2.2.1",      :require => false
 gem "elif",                           "=0.1.0",       :require => false
+gem "eventmachine",                   "=1.0.7",       :require => false
 gem "haml",                           "~>4.0.5",      :require => false
 gem 'haml-rails',                     "~> 0.4",       :require => false
 gem "inifile",                        "~>3.0",        :require => false
+gem "json",                           "=1.8.2",       :require => false
```

Comment:
Let's not add indirect dependencies to the Gemfile.  We can track this type of thing elsewhere.